app/views/command-line.html

```diff
@@ -0,0 +1,77 @@
+  </div>
+  <div class="middle">
+    <!-- Middle section -->
+    <div class="middle-section surface-shaded">
+      <div class="middle-container has-scroll">
+        <div class="middle-content">
+          <div class="container surface-shaded gutter-top">
+            <div class="row">
+              <div class="col-md-12">
+                <div class="command-line">
+                  <div class="row">
+                    <div class="col-md-2 about-icon gutter-top hidden-sm hidden-xs">
+                      <img src="images/openshift-logo.svg" />
+                    </div>
+                    <div class="col-md-9">
+                      <h1>OpenShift by Red Hat&reg;</h1>
+                      <h2 id="cli">Command Line Tools</h2>
+                      <p>
+                        With the OpenShift command line interface (CLI), you can create applications and manage OpenShift projects from a terminal.
+                        <span ng-if="cliDownloadURLPresent">
+                          You can download the <code>oc</code> client tool using the links below. For more information about downloading and installing it, please refer to the <a href="{{'get_started_cli' | helpLink}}">Get Started with the CLI</a> documentation.
+                        </span>
+                        <span ng-if="!cliDownloadURLPresent">
+                          Refer to the <a href="{{'get_started_cli' | helpLink}}">Get Started with the CLI</a> documentation for instructions about downloading and installing the <code>oc</code> client tool.
+                        </span>
+                        <div ng-if="cliDownloadURLPresent">
+                          <label class="cli-download-label">Download <code>oc</code>:</label>
+                          <div ng-repeat="(key, value) in cliDownloadURL">
+                            <a href="{{value}}" class="cli-download-link">
+                              {{key}}
+                              <i class="fa fa-external-link"></i>
+                            </a>
+                          </div>
+                        </div>
+                      </p>
+                      <p>
+                        After downloading and installing it, you can start by logging in using<span ng-if="sessionToken"> this current session token</span>:
+                        <div class="code prettyprint ng-binding" ng-if="sessionToken">
+                          oc login {{loginBaseURL}} --token=<span ng-show="showSessionToken">{{sessionToken}}</span><a href="#" ng-click="showSessionToken = !showSessionToken" ng-show="!showSessionToken">...click to show token...</a>
+                        </div>
+                        <pre class="code prettyprint ng-binding" ng-if="!sessionToken">
+        oc login {{loginBaseURL}}
+        </pre>
+                      </p>
+                      <p>
+                        <span class="pficon pficon-warning-triangle-o" style="cursor: help; vertical-align: middle;"></span>
```

Comment:
how does this look if you use the actual alert warning styles here?  this looks strange bolded to me